dds/DCPS/security/SSL/SignedDocument.cpp

```diff
@@ -28,59 +28,40 @@ namespace {
   const char* const data_filename = "<no filename: data uri>";
 }
 
-SignedDocument::SignedDocument(const std::string& uri)
-  : doc_(0)
-  , original_()
-  , verifiable_("")
-  , filename_(default_filename)
-{
-  DDS::Security::SecurityException ex;
-  if (!load(uri, ex)) {
-    ACE_ERROR((LM_WARNING, "(%P|%t) %C\n", ex.message.in()));
-  }
-}
-
 SignedDocument::SignedDocument()
-  : doc_(0)
-  , original_()
-  , verifiable_("")
+  : original_()
+  , content_()
+  , verified_(false)
   , filename_(default_filename)
 {
 }
 
 SignedDocument::SignedDocument(const DDS::OctetSeq& src)
-  : doc_(0)
-  , original_()
-  , verifiable_("")
+  : original_(src)
+  , content_()
+  , verified_(false)
   , filename_(data_filename)
 {
-  deserialize(src);
 }
 
 SignedDocument::SignedDocument(const SignedDocument& rhs)
-  : doc_(0)
-  , original_()
-  , verifiable_("")
+  : original_(rhs.original_)
+  , content_(rhs.content_)
+  , verified_(rhs.verified_)
```

Comment:
Looks like this turned in to the same as the compiler-generated copy constructor.  Could we just use that (similar for copy assignment below).